tlf/handle_test.go

```diff
@@ -241,6 +285,30 @@ func TestHandleAccessorsPublic(t *testing.T) {
 		})
 }
 
+func TestHandleAccessorsSingleTeam(t *testing.T) {
+	w := []keybase1.UserOrTeamID{
+		keybase1.MakeTestTeamID(4).AsUserOrTeam(),
+	}
+
+	h, err := MakeHandle(
+		w, nil, nil, nil, nil)
+	require.NoError(t, err)
+
+	require.Equal(t, SingleTeam, h.Type())
+
+	// All types of IsWriter/IsReader calls should panic for team TLFs.
```

Comment:
I see. That makes sense.

Although it seems better to have one place that does the check (including deciding whether we need to query service or API server, or just based on the handle itself), instead of having to do the judgement everywhere `IsWriter`/`IsReader` is called. If not `tlf`, perhaps a `tlfutils` package?